tests/app.test.ts

```diff
@@ -103,4 +144,16 @@ describe('Application Integration Tests', () => {
                 expect(e.response.status).toBe(400);
             });
     });
+
+    it('uploads a manuscript file', async () => {
+        const r1 = await startSubmission(apollo, 'researchArticle');
+        // assert good
+
+        const id = r1.data && r1.data.startSubmission ? r1.data.startSubmission.id : '';
+        expect(id).toHaveLength(36);
+        const r2 = await uploadManuscript(apollo, id);
+
+        const data = r2.data ? r2.data : {};
+        expect(data.uploadManuscript.id).toBe(id);
```

Comment:
I'm surprised this isn't complaining that `uploadManuscript` might be undefined on `data` if the turnery above is falsey.

Reply:
me too :thinking:

Reply:
mysterious :wizard: